### DISCUSSION TITLE
Broken trend arrows

### DISCUSSION THREAD

**Discussion by aptash** [NightscoutFoundation/xDrip#3163] [Q&A]
Hello!

After switching to a new phone I have a problem with some trend arrows.
Infinix Zero 20 Android 13.

For example, instead of 
![image](https://github.com/NightscoutFoundation/xDrip/assets/5793551/b859bfcf-e982-436c-ba56-104a17a8e353)
I see:
![image](https://github.com/NightscoutFoundation/xDrip/assets/5793551/7270b452-cf3e-491c-8574-d10329acdc86)

Also I have a problem with this kind of arrows:
![image](https://github.com/NightscoutFoundation/xDrip/assets/5793551/a386f3c5-d1dc-499b-bf20-490d264049a4)

I tried to change system fonts but without any success.
It's important to mention that Aaps have the same problem but not everywhere:
![image](https://github.com/NightscoutFoundation/xDrip/assets/5793551/4d500475-77a9-4bd7-9fba-a86e2d8e7969)
and the problem for the same arrow in another place:
![image](https://github.com/NightscoutFoundation/xDrip/assets/5793551/e1054f1f-778f-471b-bfe7-b088000e92cf)

Any ideas about how to fix it are welcome!
Thanks in advance!

**Comment by Navid200** [comment]
You are showing images as the description of the problem.  Can you explain what you expect to see instead of what you have shown?  Without that, you are leaving it to the imagination of the reader.

**Reply by aptash** [reply]
First of all thanks a lot for your attention!

Instead of `8,5'n` I want to see `8,5->`

![image](https://github.com/NightscoutFoundation/xDrip/assets/5793551/0bdc1e5a-5413-4a69-bebf-4dcf0315dae6)

**Reply by Navid200** [reply]
I could be wrong.  But, the only time I don't see an arrow is when I have only had one reading.  When do you see that?  All the time?

**Reply by Navid200** [reply]
Oh, you mean it shows 'n instead of an arrow!

**Reply by Navid200** [reply]
What version of xDrip are you using?

**Reply by aptash** [reply]
Yes, I see `'n` instead of an `->` arrow all the time. No matter how many readings I have.

I always use the last stable version and update it as xDrip offers an update.
But the problem occurred for the first time as soon as I migrated from an old phone to a new one.

**Reply by Navid200** [reply]
Do you see this reported by other users of that phone in public forums?

**Reply by aptash** [reply]
No. I didn't found anyone mention such problem.
Also my phone isn't very popular.

**Reply by Navid200** [reply]
Have you found a solution?
If not, did you ever try different fonts?

**Reply by Navid200** [reply]
And what about graphical trend arrow?  Does that solve the problem?
https://navid200.github.io/xDrip/docs/Display/TrendArrow.html

**Reply by aptash** [reply]
> Have you found a solution?

No.

> If not, did you ever try different fonts?

Yes. Without any success.



> And what about graphical trend arrow?

I don't try it.

Here is how it looks today for ↑↑:
![5368692035076351009](https://github.com/user-attachments/assets/bfc68a3b-c0e0-4479-a5db-07ae333f5e2f)

**Reply by Navid200** [reply]
Do you want to try the graphical arrow?
It should fix the problem.  https://navid200.github.io/xDrip/docs/Display/TrendArrow.html
